### PR TITLE
Mvs 578 update docker compose

### PR DESCRIPTION
**Pull request by jensen0914**
## :floppy_disk: [Code Check]
_Make sure you've checked/done the following before making this PR!_
* Does your code build/run?
* Does your design approach make sense?
* Does your work achieve the intention of the card?
* Does your code have any unintentional side effects -- have you tested a full workflow & checked the logic branches you touched?
* Did you make a unit test for your code (as appropriate)?
* Is your branch relatively up to date with the Development branch (within ~2 commits)
* Have you moved the card to "Needs QA" in Jira

## :flipper: [JIRA Sprint Card]

https://mass-immunization-system.atlassian.net/browse/MVS-578

## :newspaper: [Work Description]

- Added a sandbox directory to spin up all the required components for a MassVaxx system so it can run on one Docker host server. 
- Also updated main README.md in root to deprecate former Docker setup and provide a way to run system sing Node.js without Docker mostly for front-end development.

## :vertical_traffic_light: [Testing/QA Notes]

To test, following instructions provided in README.md (for dev setup using Node.js) and sandbox/README.md (for full setup using Docker).

### CODE REVIEW

File: sandbox/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    # It will also handle enabling https (and forcing http to https).
+    #
+    nginx: 
+        image: nginx:stable-alpine
+        volumes:
+            - ${PROXY_VOLUME_MAP1}
+            - ${PROXY_VOLUME_MAP2}
+            - ${PROXY_VOLUME_MAP3}
+        ports:
+            - "80:80"
```
Comment (rrucker3) [inline sandbox/docker-compose.yml:115]: Is there a way to change which port docker-compose is using for something? I have IIS running on my computer, so I can't bind port 80 to something else (or don't know how to).

File: sandbox/README.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+## Steps:
+1. In sandbox directory, generate certificates for https connections (Windows users may need
+to install an OpenSSL binary):
+
+        openssl req -newkey rsa:2048 -nodes -keyout sandbox.key -x509 -days 365 -out sandbox.crt
+
+2. In sandbox directory, copy **env.template** to a file named **.env** (note the dot in .env filename).
+3. In sandbox directory, copy **hapi.properties.template** to a file named **hapi.properties**.
+4. Protect .env and hapi.properties files to allow only Docker host root/admin user access.
```
Comment (rrucker3) [inline sandbox/README.md:20]: Is this done through right click -> properties? I'm not sure how to do this.

**Review by rrucker3 — COMMENTED** [review]
I wasn't able to get it to run because I'm using port 80 for something else. Is there a way to change which port is exposed by docker-compose?

File: sandbox/docker-compose.yml (hunk 1 above)
Comment (rrucker3) [reply sandbox/docker-compose.yml:115]: From Todd: "You could copy the docker.compose.yml to something like docker-compose-rob.yml, edit the "80:80" and "443:443" mappings in the  to change the external port to what you would like, e.g., "81:80" and then run docker-compose -f docker-compose-rob.yml -p massvaxx up"

**Review by rrucker3 — APPROVED** [review]

**Review by david-fraley — APPROVED** [review]

**Review by david-fraley — APPROVED** [review]

**Pull request merged**